hyperspy/tests/io/test_bruker.py

```diff
@@ -83,56 +80,70 @@ def test_hyperspy_wrap():
-                                                                      'azimuth_angle': 90.0,
-                                                                      'real_time': 328.8}},
-                                                 'magnification': 131.1433,
-                                                 'Stage': {'tilt_alpha': 0.5,
-                                                           'rotation': 329.49719,
-                                                           'x': 62409.2,
-                                                           'y': 36517.61,
-                                                           'z': 40234.7}}, },
-              'General': {'title': 'EDX',
-                          'time': '17:05:03',
-                          'original_filename': 'P45_instructively_packed_16bit_compressed.bcf',
-                          'date': '2016-04-01'},
-              'Signal': {'binned': True,
-                         'quantity': 'X-rays (Counts)',
-                         'signal_type': 'EDS_SEM'}}
-
-    md_ref['General']['original_filename'] = hype.metadata.General.original_filename
+    md_ref = {'Acquisition_instrument': {
+        'SEM': {
+            'Detector': {
+                'EDS': {
+                    'azimuth_angle': 90.0,
+                    'detector_type': 'XFlash 6|10',
+                    'elevation_angle': 35.0,
+                    'energy_resolution_MnKa': 130.0,
+                    'real_time': 73.882112}},
+            'Stage': {
+                'rotation': 325.54632,
+                'tilt_alpha': 0.0,
+                'x': 67358.85,
+                'y': 62116.44,
+                'z': 38818.15},
+            'beam_energy': 20,
+            'magnification': 3138.54272}},
+        'General': {
+            'date': '2018-09-12',
+            'original_filename': '16x16_instructively_packed_16bit_compressed.bcf',
+            'time': '13:01:53',
+            'title': 'EDX'},
+        'Sample': {
+            'elements': ['Al', 'C', 'Ca', 'Ce', 'Cr', 'Dy', 'Fe', 'Gd', 'La',
+                         'Mg', 'Mn', 'Nb', 'Nd', 'O', 'Si', 'Sm', 'Sr', 'Ti',
+                         'V',  'Y', 'Zr'],
+            'name': 'Map data',
+            'xray_lines': ['Al_Ka', 'C_Ka', 'Ca_Ka', 'Ce_La', 'Cr_Ka',
+                           'Dy_La', 'Fe_Ka', 'Gd_La', 'La_La', 'Mg_Ka',
+                           'Mn_Ka', 'Nb_La', 'Nd_La', 'O_Ka', 'Si_Ka',
+                           'Sm_La', 'Sr_La', 'Ti_Ka', 'V_Ka', 'Y_La',
+                           'Zr_La']},
+        'Signal': {
+            'binned': True,
+            'quantity': 'X-rays (Counts)',
+            'signal_type': 'EDS_SEM'},
+        '_HyperSpy': {
+            'Folding': {'original_axes_manager': None,
+            'original_shape': None,
+            'signal_unfolded': False,
+            'unfolded': False}}}
+
+    #md_ref['General']['original_filename'] = hype.metadata.General.original_filename #NANI???
```

Comment:
Looks this was used to hack around the hyperspy metadata original_filename full path leak on windows.